.github/workflows/docker_push.yml

```diff
@@ -0,0 +1,64 @@
+# GitHub actions workflow which builds and publishes the docker images.
+
+name: Build and push docker images
+
+on:
+  push:
+    tags: ["v*"]
+    # uncomment this to enable all pushes to main
+    #branches: [ master, main ]
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up QEMU
+        id: qemu
+        uses: docker/setup-qemu-action@v1
+        with:
+          platforms: arm64
+
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Inspect builder
+        run: docker buildx inspect
+
+      - name: Log in to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKER_HUB_USERNAME }}
+          password: ${{ secrets.DOCKER_HUB_TOKEN }}
+
+      # The following line can be used to tag builds with the SHA hash of the commit.
+      # This would be useful if we wanted to re-enable building for all commits on `main`.
+      # type=sha,enable=true,priority=100,prefix=,suffix=,format=long
```

Comment:
This is what would generate the following tag I assume?

![image](https://user-images.githubusercontent.com/1342360/128879287-56334122-3cca-4596-aedd-9fa0808bcd43.png)

Reply:
Yeah, that's right. Reckon I should just cut it out entirely?

Reply:
yeah, I'd cut it.